tests/third_party/cupy/core_tests/test_ndarray_unary_op.py

```diff
@@ -124,7 +124,12 @@ def test_invert_array(self):
     @testing.numpy_cupy_allclose(accept_error=TypeError)
```

Comment:
we can extend with a tuple (here and after) to accept errors of different types:
```suggestion
    @testing.numpy_cupy_allclose(accept_error=(ValueError, TypeError))
```